kratos/containers/array_1d.h

```diff
@@ -103,29 +102,41 @@ class	array_1d	: public boost::numeric::ublas::vector_expression< array_1d<T, N>
     array_1d ():
         vector_expression<self_type> ()
     {
-        //sin esto no funciona en windows!!
-        //std::fill (data().begin(), data().end(), value_type	());
+        std::fill(data().begin(), data().end(), value_type());
```

Comment:
@KratosMultiphysics/technical-committee don't agree with this change.

We avoid this for performance reasons. If someone needs to initialize it he/she may use the ZeroVector or ScalarVector while constructing. For setting to zero one can also call the clear method.